bin/set-legal-scanning-exclusions.sh

```diff
@@ -34,3 +34,4 @@ docker-compose ${OPTIONS} exec $@ zap-proxy zap-cli exclude '.*www.google-analyt
 docker-compose ${OPTIONS} exec $@ zap-proxy zap-cli exclude '.*edgedl/chrome.*'
 docker-compose ${OPTIONS} exec $@ zap-proxy zap-cli exclude '.*authentication-web.*'
 docker-compose ${OPTIONS} exec $@ zap-proxy zap-cli exclude '.*(js|img|stylesheets)/lib.*'
+docker-compose ${OPTIONS} exec $@ zap-proxy zap-cli exclude '.*/livereload.js.*'
```

Comment:
no

Reply:
proper fix is to set NODE_ENV to dockertests like what has been done in the citizen frontend recently.